src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -152,6 +152,9 @@
 
     <ILLinkSubstitutionsXmls Include="$(ILLinkDirectory)ILLink.Substitutions.$(Platform).xml"
                              Condition="Exists('$(ILLinkDirectory)ILLink.Substitutions.$(Platform).xml')" />
+
+    <ILLinkLinkAttributes Include="$(ILLinkDirectory)/ILLink.LinkAttributes.$(Platform).xml"
```

Comment:
Nothing is using this MSBuild Item, so it isn't doing anything yet. We need to add some logic to `eng\illink.targets` to use it.